components/home/FeaturedCards.tsx

```diff
@@ -0,0 +1,193 @@
+import { faMagnifyingGlass } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useTrendingCollections } from '@reservoir0x/reservoir-kit-ui'
+import { Box, Flex, FormatCryptoCurrency, Text } from 'components/primitives'
+import Img from 'components/primitives/Img'
+import { useMarketplaceChain } from 'hooks'
+import Link from 'next/link'
+import { ActiveMintTooltip } from './ActiveMintTooltip'
```

Comment:
This isn't being used, so we can probably remove it. Looks like it's actually not being used anywhere so we can probably kill it.